### PR TITLE
Fix style for test files

### PR DESCRIPTION
**Pull request by krlmlr**
Automated and manual reformatting.

### CODE REVIEW

File: tests/testthat/helper-combine.R [hunk 1]
```diff
@@ -1,20 +1,30 @@
 combine_pair_test <- function(item_pair, var1, var2, result,
                               can_combine = TRUE, warning = FALSE)  {
-  label_if_fail <- paste0("combine(items[c(\"", var1, "\", \"", var2, "\")])")
-  warning_regexp <- if (warning) ".*" else NA
+  label_if_fail <-
```
Comment (hadley) [inline tests/testthat/helper-combine.R]: Extra new line here?

File: tests/testthat/helper-combine.R [hunk 2]
```diff
@@ -96,7 +106,10 @@ combine_result <- function(item1, item2,
       result <- factor(c(as.character(item1), as.character(item2)))
     } else {
       # Default combination result
-      result <- unlist(list(item1,item2), recursive = FALSE, use.names = FALSE)
+      result <-
```
Comment (hadley) [inline tests/testthat/helper-combine.R]: And here? Should have line break after `(`?

File: tests/testthat/test-binds.R [hunk 3]
```diff
@@ -251,8 +254,8 @@ test_that("bind_rows can handle lists (#1104)", {
 
 test_that("bind_rows keeps ordered factors (#948)", {
   y <- bind_rows(
-    data.frame(x = factor(c(1, 2, 3), ordered = TRUE)),
-    data.frame(x = factor(c(1, 2, 3), ordered = TRUE))
+    data.frame(x = factor(c(1,2,3),ordered = TRUE)),
```
Comment (hadley) [inline tests/testthat/test-binds.R]: Why is this removing spaces?

**Comment by krlmlr** [comment]
Will apply any feedback to 4982163 directly to master.

**Pull request merged**